### PR TITLE
[ACT-482] Added XcodebuildOptions input

### PR DESCRIPTION
**Pull request by Szabadember**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -42,6 +43,7 @@ type config struct {
 	ExportMethod                    string `env:"export_method,opt[none,app-store,development,developer-id]"`
 	CustomExportOptionsPlistContent string `env:"custom_export_options_plist_content"`
 
+	XcodebuildOptions         string `env:"xcodebuild_options"`
```
Comment (trapacska) [inline main.go:46]: Please have the same key and description as in xcode-archive.

**Review by trapacska — DISMISSED** [review]

File: main.go (hunk 1 above)
Comment (Szabadember) [reply main.go:46]: It is the same

**Review by lpusok — CHANGES_REQUESTED** [review]
step.yml changes seem to be missing.

**Review by lpusok — CHANGES_REQUESTED** [review]
Check the comment. (Sorry it is missing)

File: step.yml [hunk 2]
```diff
@@ -65,6 +65,15 @@ inputs:
 
         Call `xcodebuild -help` for available export options.
       category: "app/pkg export configs"
+  - xcodebuild_options:
+    opts:
+      category: Debug
```
Comment (lpusok) [inline step.yml]: Adding at the end of the ""xcodebuild configs" category would be more useful than a single element "Debug" category.

**Review by lpusok — CHANGES_REQUESTED** [review]
Added missing comment.

File: step.yml (hunk 2 above)
Comment (Szabadember) [reply step.yml]: should we also change this for the xcode-archive step?

Comment (lpusok) [reply step.yml]: No as the ios and mac export step inputs have different structure.

File: step.yml [hunk 3]
```diff
@@ -65,6 +65,15 @@ inputs:
 
         Call `xcodebuild -help` for available export options.
       category: "app/pkg export configs"
+  - xcodebuild_options:
```
Comment (lpusok) [inline step.yml]: Please move the input after *workdir*. The order counts inside a category, and this is a less important one. You can check how it looks using `bitrise :workflow-editor`.

**Review by lpusok — CHANGES_REQUESTED** [review]

File: step.yml (hunk 3 above)
Comment (Szabadember) [reply step.yml]: moved it to the end, looks good in the workflow editor

**Review by lpusok — APPROVED** [review]

**Pull request merged**